crates/samples/echomain-stateful/src/app.rs

```diff
@@ -287,8 +287,8 @@ pub struct AppInstance {
     hostname_: HSTRING,
     tx_: Cell<Option<Sender<()>>>, // hack to use this mutably
     th_: Cell<Option<JoinHandle<Result<(), Error>>>>,
-    role_: Cell<mssf_com::FABRIC_REPLICA_ROLE>,
-    replicator_: Cell<Option<IFabricReplicator>>,
+    // role_: Cell<mssf_com::FABRIC_REPLICA_ROLE>,
```

Comment:
nit: why not delete  it instead of comment..

Reply:
It will be brought back for any meaningful example.